microsoft/devhome
# Add third party notices link to Settings About

## Summary of the pull request
Adds link to view third party notices

## References and relevant issues
#2717 
https://task.ms/48892547

## Detailed description of the pull request / Additional comments
This adds the notice.txt to Assets and launches via a hyperlink on the Settings/About page. This is done in the same manner as other preinstalled applications, e.g. Photos.

![image](https://github.com/microsoft/devhome/assets/7259210/2e9159ff-9a27-40a8-b338-8910de8bde58)

## Validation steps performed
Navigate to Settings, About and click on Third Party Notices. Notice that the file is launched in the default text editor.

## PR checklist
- [ ] Closes #2717
